source/Meadow-Filter-Parse.js

```diff
@@ -160,6 +160,47 @@ const addFilterStanzaToQuery = (pFilterStanza, pQuery) =>
 			}
 			break;
 
+    case 'FBJV':// Filter by JSON Value (left-side AND)
```

Comment:
nit: this is a personal preference for `FBVJ`, so it lines up neater with the existing `FBV`.